es5/spec/database/query/equalTo.spec.js

```diff
@@ -86,6 +89,34 @@ describe(".equalTo(query)", function () {
     queryA.equalTo(queryB).should.be["false"];
   });
 
+  describe("(where)", function () {
```

Comment:
This is not the right file for this. we're not testing mocks in this file.

If anything, this should be `mock.where.spec.js`